modules/gateway/peers.go

```diff
@@ -168,7 +168,7 @@ func (g *Gateway) Connect(addr modules.NetAddress) error {
 		return err
 	}
 	// send our version
-	if err := encoding.WriteObject(conn, build.Version); err != nil {
+	if err := encoding.WriteObject(conn, "0.3.3"); err != nil {
```

Comment:
Based on how Bitcoin works, I think we can drop support for this after 95% of nodes have build.Version >= 0.3.3.2